concrete/controllers/single_page/dashboard/system/files/thumbnails.php

```diff
@@ -113,15 +149,22 @@ public function update()
             $this->error->add(t('Invalid thumbnail type object.'));
         }
         if (!$this->error->has()) {
-            $type->setWidth($request->request->get('ftTypeWidth'));
+            
             $height = intval($request->request->get('ftTypeHeight'));
+            $width = intval($request->request->get('ftTypeWidth'));
```

Comment:
Just a (really) minor performance note: `(int) $value` is much faster than `intval($value)`